### ISSUE TITLE
[MU4 Issue] [Accessibility] NVDA doesn't read menus properly

### ISSUE THREAD

**Issue by shoogle** [musescore/MuseScore#10712]
**Describe the bug**
When using MuseScore with the NVDA screen reader, menus like File and Edit are not spoken properly. Items in the menu (e.g. New, Open, Save, Export, etc.) are announced as _"Grouping"_.

![image](https://user-images.githubusercontent.com/11011881/156416693-17b986b0-1083-4e28-8752-0be53f58922d.png)

**To Reproduce**
1. Launch MuseScore and the [NVDA screen reader](https://www.nvaccess.org/download/) (not Windows Narrator).
2. Press Alt+F to open the File menu. It will open on the first item (New).
   - NVDA says _"MuseScore 4 Development Windows, Grouping"_.
   - It should say _"File menu... New, Ctrl+N"_ or similar.
4. Press the Down arrow keys to move to the second item (Open).
    - NVDA says _"Grouping"_.
    - It should say _"Open, Ctrl+O"_ or similar.
5. Press the Down arrow key again to move to the third item (Open recent).
    - NVDA again says _"Grouping"_.
    - It should say _"Open recent submenu"_ or similar.

**Expected behavior**
Menu items should be spoken as well as the shortcut used to trigger them. (E.g. _"Save, Ctrl+S"_)

**Platform information**
 - OS: Windows 10 running the NVDA screen reader. (Does not occur with Windows Narrator.)

**Additional context**
You can listen using Windows Narrator to get an idea of what expected speech output should be.

**Comment by tyrylu** [comment]
With the Musescore revision send to the A11Y testers (the revision information makes no sense, unfortunately) and NVDA snapshot alpha-25243,d113c083 when you move in the menus, there is actually no speech at all. And there's nothing interesting in the NVDA debug log either.

**Comment by shoogle** [comment]
Thanks @tyrylu! The accessibility testers are using MuseScore 4 Alpha version (revision cafefd7).

**Comment by Eism** [comment]
@shoogle 
I just tested the accessibility on Windows 11 NVDA and can't reproduce the bug
Please recheck on master

**Comment by tyrylu** [comment]
Yeah, somewhat recent master (four commits before the current as of now), I can break the menu presentation logic when using the right/left arrows when inside a menu, it sometimes exits the menu, but it is read, yes.
